tvb_epilepsy/base/model_configuration.py

```diff
@@ -3,17 +3,39 @@
 Class to keep the model configuration values.
 This will be used to populate a Model instance needed in order to launch a simulation.
 """
+
 from collections import OrderedDict
 
-from tvb_epilepsy.base.h5_model import prepare_for_h5
-from tvb_epilepsy.base.utils import formal_repr
+import numpy as np
+
 from tvb_epilepsy.base.constants import X0_DEF, K_DEF, YC_DEF, I_EXT1_DEF, A_DEF, B_DEF
 
+from tvb_epilepsy.base.utils import formal_repr, dicts_of_lists_to_lists_of_dicts
+from tvb_epilepsy.base.h5_model import convert_to_h5_model
+
+from tvb_epilepsy.base.constants import X1_DEF, X1_EQ_CR_DEF, X0_CR_DEF, \
+                                        FOLDER_FIGURES, VERY_LARGE_SIZE, SMALL_SIZE, FIG_FORMAT, SAVE_FLAG, SHOW_FLAG
+from matplotlib import pyplot
+try:
+    #https://github.com/joferkington/mpldatacursor
+    #pip install mpldatacursor
+    #Not working with the MacosX graphic's backend
+    from mpldatacursor import HighlightingDataCursor #datacursor
+    MOUSEHOOVER = True
+except ImportError:
+    pass
+
+from tvb_epilepsy.base.plot_factory import plot_in_columns, _save_figure, _check_show
+from tvb_epilepsy.base.calculations_factory import calc_fx1, calc_fz, calc_fx1_2d_taylor, calc_rescaled_x0, \
```

Comment:
If we need calculations, then that code should go into model_configuration_service, and not be in the model.